pastel/client.go

```diff
@@ -382,6 +382,27 @@ func (client *client) GetRegisterNFTFee(ctx context.Context, request GetRegister
 	return totalStorageFee.TotalStorageFee, nil
 }
 
+func (client *client) GetActionFee(ctx context.Context, actionType string, ImgSizeInMb int64) (int64, error) {
+	if actionType != "sense" && actionType != "cascade" {
+		return 0, errors.Errorf("invalid action type: %s", actionType)
+	}
+
+	var actionFee struct {
+		ActionFee int64 `json:"actionfee"`
+	}
+
+	params := []interface{}{}
+	params = append(params, "getactionfee")
+	params = append(params, actionType)
+	params = append(params, ImgSizeInMb)
```

Comment:
API was modified. call:
`getactionfees <data size>`

response:
```
{
  "sensefee": x,
  "cascadefee": y
}
```

Reply:
ok, I will improve pastel API call with the latest commit.